api/src/organizational-entities/infrastructure/repositories/organization-for-admin.repository.js

```diff
@@ -216,15 +216,9 @@ const update = async function (organization) {
     .where({ id: organization.id });
 };
 
-/**
- * @typedef {Object} OrganizationForAdminRepository
- * @property {archive} archive
- * @property {exist} exist
- * @property {findChildrenByParentOrganizationId} findChildrenByParentOrganizationId
- * @property {get} get
- * @property {save} save
- * @property {update} update
- */
+const deleteById = async function (organizationId) {
+  await knex(ORGANIZATIONS_TABLE_NAME).where({ id: organizationId }).delete();
+};
```

Comment:
Créer cette nouvelle fonction en l'équipant de base pour pouvoir être transactionnelle : 
```suggestion
const deleteById = async function (organizationId) {
  const knexConn = DomainTransaction.getConnection();
  await knexConn(ORGANIZATIONS_TABLE_NAME).where({ id: organizationId }).delete();
};
```